CMakeLists.txt

```diff
@@ -53,6 +53,12 @@ install(DIRECTORY ${CMAKE_LIBRARY_OUTPUT_DIRECTORY}/ DESTINATION ${CMAKE_INSTALL
 install(DIRECTORY pd-help/ DESTINATION ${CMAKE_INSTALL_PREFIX})
 
 #Dependencies: availability
+#Non-standard openssl path on OSX
+if(${CMAKE_SYSTEM_NAME} STREQUAL "Darwin")
+  set(OPENSSL_ROOT_DIR "/usr/local/opt/openssl")
+endif()
+find_package(OpenSSL)
```

Comment:
I think it just doesn't work with find_library. Not fluent enough in CMake to know why but I am 99% sure I tried